Libraries/D3D12RaytracingFallback/src/pch.h

```diff
@@ -91,8 +91,8 @@
 #include "UberShaderBindings.h"
 #include "UberShaderRaytracingProgram.h"
 
-#define USE_PIX_MARKERS 0
+#define USE_PIX_MARKERS 1
 #if USE_PIX_MARKERS
-#include <pix3.h>
+#include "..\..\..\packages\WinPixEventRuntime.1.0.180612001\Include\WinPixEventRuntime\pix3.h"
```

Comment:
Unfortunately I've been unable to fix FallbackLayer not directly seeing pix3.h and this will need to get updated when nugets get updated in the project.